app/bin/service/search.dart

```diff
@@ -75,6 +76,12 @@ void _main(int isolateId) {
       [
         new ManualTriggerTaskSource(taskReceivePort),
         new IndexUpdateTaskSource(db.dbService, batchIndexUpdater),
+        new DatastoreVersionsHeadTaskSource(
+          db.dbService,
+          TaskSourceModel.analysis,
+          sleep: const Duration(minutes: 10),
```

Comment:
So every 10 minutes, what happens?

Reply:
we poll the datastore to see if there are any new `Analysis` instances created